extensions/runtime-metrics/src/main/java/io/opentelemetry/extension/metrics/runtime/MemoryPools.java

```diff
@@ -70,30 +68,20 @@ public MemoryPools() {
 
   /** Export only the "area" metric. */
   public void exportMemoryAreaMetric() {
-    final LongUpDownSumObserver areaMetric =
-        this.meter
-            .longUpDownSumObserverBuilder("runtime.jvm.memory.area")
-            .setDescription("Bytes of a given JVM memory area.")
-            .setUnit("By")
-            .build();
-    areaMetric.setCallback(
```

Comment:
Awesome!

Reply:
obviously, we'll want to port this change over to the instrumentation repo, since we're deprecating this code. Mostly did this to verify that it was a viable usage pattern. :)